README.md

```diff
@@ -7,7 +7,7 @@ It provides an API to [job-runner](https://github.com/opensafely-core/job-runner
 ## Stack
 
 This is a [Django](https://www.djangoproject.com) project.
-It uses [Django Rest Framework](https://www.django-rest-framework.org) for the API and [SQLite](https://www.sqlite.org/index.html) for the database.
+It uses [Django Rest Framework](https://www.django-rest-framework.org) for the API and [PostgreSQL](https://www.postgresql.org/) for the database.
```

Comment:
Do we need brief instructions for how to set up a local postgres database?

Reply:
I think so yes, I'll add that in a subsequent PR since it will need to touch a couple of different OSen.